MapboxCoreNavigation.podspec

```diff
@@ -40,6 +40,7 @@ Pod::Spec.new do |s|
   s.requires_arc = true
   s.module_name = "MapboxCoreNavigation"
 
+  s.dependency "MapboxNavigationNative", "~> 3.0"
```

Comment:
Bump to 3.1.2 which includes Bitcode and various fixes.

Reply:
@frederoni It's actually resolving to 3.1.3. 

```diff
-binary "https://s3.amazonaws.com/mapbox/mapbox-navigation-native/ios/builds/Mapbox-iOS-Navigation-Native.json" "3.1.2"
+binary "https://s3.amazonaws.com/mapbox/mapbox-navigation-native/ios/builds/Mapbox-iOS-Navigation-Native.json" "3.1.3"
```